src/main/scala/za/co/absa/hyperdrive/trigger/api/rest/services/WorkflowValidationService.scala

```diff
@@ -82,4 +84,61 @@ class WorkflowValidationServiceImpl @Inject()(override val workflowRepository: W
     }
     Future.successful(projectValidation.toSeq)
   }
+
+  private[services] def validateWorkflowData(originalWorkflow: WorkflowJoined, updatedWorkflow: WorkflowJoined)(implicit ec: ExecutionContext): Future[Seq[ApiError]] = {
```

Comment:
any specific reason for injecting services in these methods?

Reply:
I am not injecting services.
private[services] allows me to test this function.
Otherwise this method will be not accessible